storage/storage_test.go

```diff
@@ -939,6 +939,11 @@ var lsblkOutput string = `{
             {"name": "sdg3", "path": "/dev/sdg3", "size": "8G",    "type": "part", "fstype": "ext4", "partlabel": "CLR_ROOT", "mountpoint": "/"},
             {"name": "sdg4", "path": "/dev/sdg4", "size": "8G",    "type": "part", "fstype": "ext4", "partlabel": "CLR_MNT_/home", "mountpoint": "/home"}
          ]
+      },
+	  {"name": "sdh", "path": "/dev/sdh", "size": "2.0T", "type": "disk", "mountpoint": null,
```

Comment:
Not sure why this test? Is this to show we have a root partition and no explicit boot partition(simulating a legacy mode) and test this passes?